apps/src/templates/manageStudents/ManageStudentsLoginInfo.jsx

```diff
@@ -63,15 +87,24 @@ class ManageStudentsLoginInfo extends Component {
                 students={this.props.studentData}
               />
             </div>
+            <GlobalRegionWrapper
+              component={() => (
+                <SafeMarkdown
+                  markdown={i18n
+                    .setUpClass3({
+                      parentLetterLink: teacherDashboardUrl(
+                        sectionId,
+                        '/parent_letter'
+                      ),
+                    })
+                    .replace(/^\d./, `${++counter}.`)}
+                />
+              )}
+              componentId="ClassroomSetUpStep3"
+            />
```

Comment:
Rewrites the "Steps to login / join" instructions such that they remain in numerical order (1., 2., 3., etc)

Reply:
They optionally appear based on the presence of keys in the region configuration.